deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,64 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+/*Schrijf hier jouw css*/
+
+
+body{
+    font-family: 'Poppins', sans-serif;
+    background-color: #f2f3d9;
+    text-align: center;
+    margin: 0;
+}
+
+nav{
+    display: flex;
+    flex-direction: row;
+    background-color: #030027;
+    color: #E6EAEB;
+}
+
+ul{
+    display: flex;
+    list-style: none;
+    flex-direction: row;
+    flex-basis: 1500px;
+}
+.li-1{
+    display: flex;
+    flex-basis: 700px;
+}
+.div-1{
+    display: flex;
+    justify-content: space-around;
+    flex-basis: 700px;
+}
+header{
+    margin: 40px;
+}
+.main-6{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    height: 350px;
+
+}
+.art-1{
```

Comment:
Ik zie dat je per article een aparte class hebt aangegeven.

Zoals je ziet pas je nu alles 3x hetzelfde toe. Om herhaling te voorkomen kan je voor de article ook 1 unieke class name gebruiken en dan alles in één keer stylen. 

Dit scheelt een hoop regels en herhaling.